src/main.rs

```diff
@@ -485,6 +497,35 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
+        #[cfg(any(target_os = "macos", target_os = "linux"))]
+        let tray_img =
+            Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qed");
```

Comment:
Right now this is the problem:
<img width="1174" alt="image" src="https://github.com/subspace/space-acres/assets/16472948/c2def394-d2fb-4710-91e8-90b43724c75a">

The `from_png_bytes` function is not supported on `macOS`. And for `from_rgba` function, it requires different args. Don't know the purpose much.

Tried with this modification on top of latest commit so far:
```rust
        #[cfg(target_os = "linux")]
        let tray_img =
            Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qed");
        #[cfg(target_os = "macos")]
        let tray_img = Icon::from_rgba(
            TRAY_ICON.to_vec(),
            about_dialog.width() as u32,
            about_dialog.height() as u32,
        )
        .expect("Tray icon exists and is a valid PNG; qed");
```

Reply:
i will try tomorrow, if okay in linux, i will both use from rgba. but why use dialog width and height

Reply:
> but why use dialog width and height
Idk, it's asked in `from_rgba` function, so I placed some window's dimension. May be you can decide better what to parse in there.